docs/data/date-pickers/base-concepts/base-concepts.md

```diff
@@ -29,6 +29,51 @@ import { DatePicker } from '@mui/x-date-pickers';
 import { DatePicker } from '@mui/x-date-pickers-pro';
 ```
 
+## Date library
```

Comment:
Moving this section to the base concepts page to ensure a more logical flow of the information. 

I think it would make sense to create a documentation page at some point, detailing the use of adapters and date libraries.

Reply:
Our assumption when creating the Overview + Getting Started + Base concept pages was that people would see Overview before starting to use the components and thus we wanted to put inside it things that could convince people to use our app.
The fact that our packages supports several date libraries is a selling point, so I think it's sad not to have it mentioned anywhere before people actually start using the product.

With that being said, the new Overview page is not suited for a very text-oriented section like this one.
But if we have a way to add a section showing how we use several date library it would be great. Maybe with a code toggle between the 4 date libraries and a code snippet + demo preview showing how the results remains the same.

Reply:
> But if we have a way to add a section showing how we use several date library it would be great.  

That makes sense! I will think of a design to add the information that we support multiple libraries here, and we can add a 'more info' button to link to the base concepts page. WDYT?

Reply:
That would be great :tada:

Reply:
@samuelsycamore I created a section here briefly mentioning the date libraries here and moved the technical info to base concepts. But I also created an issue to separate these concepts to a different page -> https://github.com/mui/mui-x/issues/13763